File: interfaces/src/test/java/org/codroid/interfaces/utils/Description.java
package org.codroid.interfaces.utils;

import java.util.List;

public class Description {
    String name;
    long version;
    List<Long> list;
    Theme theme;

    public Description(String name, long version, List<Long> list, Theme theme) {
        this.name = name;
        this.version = version;
        this.list = list;
        this.theme = theme;
    }

    static class Theme {
        String name;
        long version;
    }
}

File: interfaces/src/main/kotlin/org/codroid/interfaces/utils/Description.kt
package org.codroid.interfaces.utils

import org.codroid.interfaces.addon.OptionalField

data class Description(
    val name: String,
    val `package`: String,
    val enterPoint: String,
    val author: String,
    val versionCode: Long,
    val versionDes: String,
    val supportVersion: String,
    val description: String,
    val link: String,
    @OptionalField val events: List<String>? = null,
    @OptionalField val theme: String? = null
)